notebooks/data_v2

```diff
@@ -0,0 +1 @@
+/data/users/fris/data_v2
```

Comment:
This doesn't seem to be pointing to anything

Reply:
thanks, we could do with cleaning that up but probably should be swept up with other fixes? Ideally we don't want paths like this in the notebooks anyway